src/unix/linux_like/linux/mod.rs

```diff
@@ -5877,6 +5877,17 @@ f! {
     pub fn ELF64_R_INFO(sym: Elf64_Xword, t: Elf64_Xword) -> Elf64_Xword {
         sym << (32 + t)
     }
+
+    #[cfg(target_env = "gnu")]
+    #[allow(unused_unsafe)]
+    pub fn SUN_LEN(s: crate::sockaddr_un) -> usize {
+        offset_of!(crate::sockaddr_un, sun_path) + crate::strlen(s.sun_path.as_ptr())
+    }
```

Comment:
Could you add `#[allow(unused_unsafe)]` in the macro instead? There isn't any `unsafe` here.